TLM/.editorconfig

```diff
@@ -87,7 +85,8 @@ indent_size=4
 
 [{*.csproj,*.ccproj,*.androidproj}]
 indent_style=space
-indent_size=4
+indent_size=2
```

Comment:
Indent size 4
Tab width 8

Reply:
tab should be same as indent, both set to `4`?